### PR TITLE
Audio service test

### PR DESCRIPTION
**Pull request by Koolaidman64**
Working test using sinon

### CODE REVIEW

File: main-app/tests/audio-tests/audioServiceTests.js [hunk 1]
```diff
@@ -1,18 +1,39 @@
...
+(function() {
     'use strict';
-    describe('AudioService Tests', function(){
-        var $document;
-        var $timeout;
+    describe('AudioService Tests', function() {
+        var sandbox;
+        var audioService;
         var audioSprite;
+        var audioSpriteMock;
```
Comment (DeclanT) [inline main-app/tests/audio-tests/audioServiceTests.js]: theres no need to use var to declare all of these, just state it the once and seperate the others with a comma. eg.
var sandbox,
      audioService;

File: main-app/tests/audio-tests/audioSpriteTests.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+
+(function() {
+    'use strict';
+    describe('AudioSprite tests', function () {
+
+        var $timeout;
+        var $document;
+        var audioSprite;
```
Comment (DeclanT) [inline main-app/tests/audio-tests/audioSpriteTests.js]: as above

File: main-app/tests/controller-tests.js [hunk 3]
```diff
@@ -7,7 +7,7 @@
         var sandbox;
         var q;
         var gameModelMock;
-        var audioMock;
+        var audioServiceMock;
```
Comment (DeclanT) [inline main-app/tests/controller-tests.js]: as above

**Comment by keithbarrow** [comment]
I've picked your PR ... up

File: main-app/tests/audio-tests/audioServiceTests.js [hunk 4]
```diff
@@ -1,18 +1,33 @@
...
+(function() {
     'use strict';
-    describe('AudioService Tests', function(){
-        var $document;
-        var $timeout;
-        var audioSprite;
+    describe('AudioService Tests', function() {
+        var sandbox,
+         audioService,
+         audioSprite,
```
Comment (keithbarrow) [inline main-app/tests/audio-tests/audioServiceTests.js]: audioSprite isn't used - remove please.

**Pull request merged**